README.md

```diff
@@ -326,7 +335,91 @@ vt = VirusTotalApi("<VIRUSTOTAL-API-KEY-HERE>", cache_file_name="/tmp/cache.viru
 ```
 
 
-#### File hash reports
+
+#### Domain report endpoint
+
+Calls `domain/report` VirusTotal API endpoint.
+Pass a list or any other Python enumerable containing the domains:
+
+```python
+domains = ["google.com", "baidu.com", "bibikun.ru"]
+vt.get_domain_reports(domains)
+```
+
+will result in:
+
+```
+{
+  "baidu.com": {
+    "undetected_referrer_samples": [
+      {
+        "positives": 0,
+        "total": 56,
+        "sha256": "e3c1aea1352362e4b5c008e16b03810192d12a4f1cc71245f5a75e796c719c69"
+      }
+    ],
+    
+    ..
+
+    }
+}
+```
+
+
+#### URL report endpoint
+
+Calls `url/report` VirusTotal API endpoint.
+Pass a list or any other Python enumerable containing the URL addresses:
+
+```python
+urls = ["http://www.google.com", "http://www.yelp.com"]
+vt.get_url_reports(urls)
+```
+
+will result in:
+
+```
+{
+  "http://www.google.com": {
+    "permalink": "https://www.virustotal.com/url/dd014af5ed6b38d9130e3f466f850e46d21b951199d53a18ef29ee9341614eaf/analysis/1423344006/",
+    "resource": "http://www.google.com",
+    "url": "http://www.google.com/",
+    "response_code": 1,
+    "scan_date": "2015-02-07 21:20:06",
+    "scan_id": "dd014af5ed6b38d9130e3f466f850e46d21b951199d53a18ef29ee9341614eaf-1423344006",
+    "verbose_msg": "Scan finished, scan information embedded in this object",
+    "filescan_id": null,
+    "positives": 0,
+    "total": 62,
+    "scans": {
+      "CLEAN MX": {
+        "detected": false,
+        "result": "clean site"
+      },
+    }
+  ..
+
+}
+```
+
+ 
+#### URL scan endpoint
+
+Calls url/scan VirusTotal API endpoint.
```

Comment:
"url" here and in the next line should be capitalized to keep the consistency with the section title.